packages/host/src/QueryErrorHandler.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect } from 'react';
+import { useNotification } from '@common/hooks';
+import { useTranslation } from '@common/intl';
+import { useQueryClient } from 'react-query';
+
+export const QueryErrorHandler = () => {
+  const client = useQueryClient();
+  const { error } = useNotification();
+  const t = useTranslation();
+
+  const generalError = t('error.general-query-error');
+
+  useEffect(() => {
```

Comment:
this is because I want to use toast notifications.. and the notifications are using hooks which I cannot access outside of a component or hook. Otherwise this would just be at the top of `Host`
It also means that I can `useTranslation`, though I expect we won't ever hit the general error message anyway..

The toasting also requires being under the `SnackbarProvider` so I had to add it lower down the stack than just in `Host`